ctlcmd/motioncmd.py

```diff
@@ -472,9 +472,9 @@ def run(self, args):
       s4 = self.visual.get_latest().s4
       sample_time = time.time_ns()
       timestamp = (sample_time - start_time) / 1e9
-      self.write_standard_line((lumival, uncval, s2, s4),
-                               det_id=-100,
-                               time=timestamp)
+      print("run self.fillroot in timescan after this")
+      self.fillroot({"lumival":lumival,"uncval":uncval,"S2":s2,"S4":s4},time=timestamp,det_id=-100)
       self.pbar_data(Lumi=f'{lumival:.2f}+-{uncval:.2f}',
                      Sharp=f'({s2:.1f}, {s4:.1f})')
       time.sleep(args.interval)
+    self.dumprootdata()
```

Comment:
Consider moving `self.dumprootdata()` to an overloaded `rootfilecmd.post_run` method [1]. This ensures that the dump will occur, even if the command is terminated earlier by user inputs

[1] https://github.com/UMDCMS/SiPMCalibControl/blob/master/ctlcmd/cmdbase.py#L577